ChangeLog.md

```diff
@@ -12,6 +12,7 @@ fixes, check out the
 ## [2.1.0] - 2021-05-16
 ### Added
  - Custom function logging targets.
+ - Localization for Spanish (es-ES :mx)
```

Comment:
I believe the flag character needs at least a trailing colon, or perhaps should be `:mexico:` [instead](https://emojipedia.org/flag-mexico/)? As is, it does not render as a flag.

Also, in addition to updating the Changelog, consider updating the Key Features section of the [README](https://github.com/goatshriek/stumpless#key-features) with the new flag as well!

Reply:
Thanks. I'll change the tag to :mexico: and add it to README.md

Reply:
but (es-ES) means Spain spanish, a think that the tag would be :es:,  because :mexico: would be (es-MX). The translation is the same, so there will be no problem with it

Reply:
Ah, great point!